extensions/amp-story/amp-story-analytics.md

```diff
@@ -153,6 +153,29 @@ The `story-page-attachment-enter` trigger is fired when a page attachment is ope
 
 The `story-page-attachment-exit` trigger is fired when a page attachment is dismissed by the user.
 
+### Story shopping product listing page view trigger (`"on": "story-shopping-plp-view"`)
+
+The `story-shopping-plp-view` trigger is fired when whenever amp-story-shopping-attachment displays the Product Listing Page without an active product.
```

Comment:
component names are always in back-ticks amp-story-shopping-attachment => `amp-story-shopping-attachment`